server/container_create.go

```diff
@@ -249,6 +250,19 @@ func setupContainerUser(ctx context.Context, specgen *generate.Generator, rootfs
 			}
 			specgen.AddMount(mnt)
 		}
+		if shadowPath != "" {
+			log.Infof(ctx, "Mounting custom /etc/shadow path")
```

Comment:
can these be debug instead?

Reply:
(this is probably not ready for review yet, is it :see_no_evil: )

Reply:
Yep, I'll plan to convert them to debug once this PR goes into a final version.